README.md

```diff
@@ -43,23 +43,31 @@ Here is what the plugin will do depending on your `fork` settings:
 
 ### Configuring
 
+For Play Framework 2.7 projects add the `sbt-aspectj-runner-play-2.7` to your `project/plugins.sbt` file:
+
+```scala
+resolvers += Resolver.bintrayIvyRepo("kamon-io", "sbt-plugins")
+addSbtPlugin("io.kamon" % "sbt-aspectj-runner-play-2.7" % "1.1.2")
+
+```
+
 For Play Framework 2.6 projects add the `sbt-aspectj-runner-play-2.6` to your `project/plugins.sbt` file:
 
 ```scala
 resolvers += Resolver.bintrayIvyRepo("kamon-io", "sbt-plugins")
-addSbtPlugin("io.kamon" % "sbt-aspectj-runner-play-2.6" % "1.1.1")
+addSbtPlugin("io.kamon" % "sbt-aspectj-runner-play-2.6" % "1.1.2")
 
 ```
 
 For Play 2.4 and 2.5 you can use the older `sbt-aspectj-play-runner` plugin:
 
 ```scala
 resolvers += Resolver.bintrayIvyRepo("kamon-io", "sbt-plugins")
-addSbtPlugin("io.kamon" % "sbt-aspectj-play-runner" % "1.0.4")
+addSbtPlugin("io.kamon" % "sbt-aspectj-play-runner" % "1.2.2")
```

Comment:
Shouldn't this one be `1.1.2` as well?

Reply:
I was wrong about this one, it has to be `1.0.4` because that is an old version that we are not releasing any more.

It should remain like this:
```suggestion
addSbtPlugin("io.kamon" % "sbt-aspectj-play-runner" % "1.0.4")
```